kumascript/macros/PreviousMenuNext.ejs

```diff
@@ -6,17 +6,15 @@ Parameter
     $1 (second parameter): path of Next page
     $2 (third parameter): path to main menu of module
 
-Issue
-    * Problem of Apostrophe (https://developer.mozilla.org/fr/docs/JavaScript_Guide/Op%C3%A9rateurs/Op%C3%A9rateurs_sp%C3%A9ciaux)
-
```

Comment:
Did you solve this issue, or just notice that it's no longer a problem?

Reply:
We currently use a unified slug for all locales (that is, the url itself will not be localized), and I think we don't have this problem now.